pai-management/doc/etcd.md

```diff
@@ -0,0 +1,59 @@
+# Etcd maintenance
+
+## Goal
+Etcd is a service used by Kubernetes as a consistent and highly-available key value store for all its backing cluster data.
+
+## Build
+OpenPAI doesn't directly build etcd service. The service's image is directly pulled from the docker registry.
+
+## Configuration
+
+Configuration file [kubernetes-configuration.yaml](../../cluster-configuration/kubernetes-configuration.yaml) defines etcd as kubernetes storage and specifies the version.
+- storage-backend: etcd3
+- etcd-version: 3.2.17
+
+## Deployment
+
+Etcd is deployed as a component of Kubernetes when running this command:
+```bash
+python paictl.py cluster k8s-bootup -p ./path/to/cluster/configuration/dir
+```
+
+## Upgrading and rolling back
+
```

Comment:
suggest tell user how to identify etcd is at which node

Reply:
add more instructions to identify the etcd nodes and verify its status. done.